DHL/JSON/9df77954-633e-4a19-8e78-a2e73e34050f.json

```diff
@@ -0,0 +1,998 @@
+                                      "Nm": "Rule condition for Shipment with DHL",
+                                      "Cond": "**.Att[Id='75e0a595-ab28-4e91-9949-fab52cdf82b5'].Val='Opt2'",
+                                      "AttrMtdt": [
+                                        {
+                                          "isMan": true,
+                                          "isHidden": false
+                                        }
+                                      ]
+                                    }
+                                  ]
+                                },
+                                {
+                                  "Id": "c33d6bc3-73e7-4c27-82ea-1a344f95a726",
+                                  "Nm": "Purpose Name - Radio button",
+                                  "Dsc": "Purpose Name with a radiobutton component",
+                                  "AttTy": "Constant",
+                                  "DflVal": "Agree to export this shipment with DHL Express",
+                                  "isExcl": false,
+                                  "isMan": false,
+                                  "Rul": [
+                                    {
+                                      "RulId": "fd16b4ee-1b53-422a-a211-4e241795bcd6",
+                                      "Nm": "Not agree purpose",
+                                      "Cond": "**.Att[Id='917a20fa-f163-454d-b941-fbe3bb21018c'].Val='Opt2'",
+                                      "AttrMtdt": [
+                                        {
+                                          "DflVal": "Not agree to export this shipment with DHL Express"
+                                        }
+                                      ]
+                                    }
+                                  ]
+                                },
+                                {
+                                  "Id": "6cfb1e3b-b505-4195-b811-3513abe6f5b6",
+                                  "Nm": "Purpose Description - Radio button",
+                                  "Dsc": "Purpose Description with a radiobutton component",
+                                  "AttTy": "Constant",
+                                  "DflVal": "Not agree to export this shipment with DHL Express",
+                                  "isExcl": false,
+                                  "isMan": false,
+                                  "Rul": [
+                                    {
+                                      "RulId": "e95ba789-9599-48a3-b975-c87af6e6a848",
+                                      "Nm": "Not agree Purpose Description",
+                                      "Cond": "**.Att[Id='917a20fa-f163-454d-b941-fbe3bb21018c'].Val='Opt1'",
+                                      "AttrMtdt": [
+                                        {
+                                          "DflVal": "Agree to export this shipment with DHL Express"
```

Comment:
user does not agree to export this shipment with DHL Express

Reply:
Done